### PR TITLE
Updating spring streaming app to Spring Boot 3.2.

### PR DESCRIPTION
**Pull request by gm2552**
Addressing issue #400

### CODE REVIEW

File: spring-streaming-app/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>3.1.6</version>
+		<version>3.2.0</version>
```
Comment (trisberg) [inline spring-streaming-app/pom.xml:8]: You don't want 3.2.1?

Comment (gm2552) [reply spring-streaming-app/pom.xml:8]: I thought 3.2.1 was out, but I didn't see it out in maven:

https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot/

Comment (gm2552) [reply spring-streaming-app/pom.xml:8]: Hmmm... it doesn't show up in that list, but I can hit it directly:

https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot/3.2.1/

Comment (gm2552) [reply spring-streaming-app/pom.xml]: OK, It's been updated to 3.2.1.  @trisberg

Comment (trisberg) [reply spring-streaming-app/pom.xml]: Maybe the repo index is outdated - it is listed here https://search.maven.org/artifact/org.springframework.boot/spring-boot-starter

**Review by trisberg — APPROVED** [review]

**Pull request merged**